app/interceptors/ExceptionInterceptor.java

```diff
@@ -72,7 +71,7 @@ private int getStatusCode(final Throwable throwable) {
     private String getMessage(final Throwable throwable, final int status) {
         String message = throwable.getMessage();
         if (StringUtils.isBlank(message)) {
-            message = HttpStatus.getStatusText(status);
+            message = Integer.toString(status);
```

Comment:
Actually I'm able to remove the dependency to 'httpcore' altogether.  See the updated pull request.  If we use org.eclipse.jetty.http.HttpStatus.getMessage(int statusCode) or the Spring one, we will have to add one more package to the code base.  I am not sure it's worth it.